sites/fast-website/webpack.common.js

```diff
@@ -4,9 +4,11 @@ const path = require("path");
 const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const FaviconsWebpackPlugin = require("favicons-webpack-plugin");
```

Comment:
We should not need this now that the `favicon.ico` file is now on CDN (https://static.fast.design/assets/manifest/favicon.ico)

Reply:
Thank you for the comment. I have removed the FaviconsWebpackPlugin and added a link to the CDN in the index.html.